content/en/blog/2025/ambient-multicluster/index.md

```diff
@@ -0,0 +1,106 @@
+
+### Service discovery and scope
+
+Once we have securely connected our clusters, we enable cross-cluster communication for a service by marking it global.
+When a service becomes global, istiod will configure east-west gateways to accept and route traffic destined to the global service.
+Istiod will also read remote apiservers and configure ztunnel with the number of pods for the global service per remote cluster.
+Ztunnel can then proxy traffic to the global service across clusters.
+
+Mesh administrators define the label-based criteria for global services via the `ServiceScope` API,
+and app developers opt into global behavior by labeling their services accordingly.
+The default `ServiceScope` is
+
+{{< text yaml >}}
+=======
+serviceScopeConfigs:
+  - servicesSelector:
+      matchExpressions:
+        - key: istio.io/global
+          operator: In
+          values: ["true"]
+    scope: GLOBAL
+{{< /text >}}
+
+meaning that any service with the `istio.io/global=true` label is global.
+Although the default value is straightforward, the API is flexible and can express complex conditions using a mix of ANDs and ORs.
+
+By default, ztunnel will load balance traffic uniformly across all endpoints --even remote ones--, but this can be configured using the service's `trafficDistribution` field to only cross cluster boundaries when there are no local endpoints.
+Thus, users have control over whether and when traffic crosses cluster boundaries.
+
+## Limitations and Roadmap
+
+Although the current implementation of ambient multicluster has the foundational features for a multicluster implementation,
+there is still a lot of work to be done.
+
+For example, currently, we require that global services, attached waypoints, and `ServiceScope` configuration have uniform configuration across all clusters.
+This greatly simplified our alpha implementation, but we are looking to allow for more configuration skew.
+
+Similarly, waypoints and L7 policy enforcement have proven difficult since different clusters might have different policies.
+In our alpha implementation, if a service has a waypoint, it will go through said waypoint in the destination cluster.
+This reduces unexpected surprises by enforcing the destination cluster's L7 authorization policy, but remove the ability to perform L7 cross-cluster failover.
+Eventually, we would like to apply L7 policy in both the source and destination cluster.
+We are also looking to improve our reference documentation, guides, testing, and performance.
+Currently, we only support a multi-primary deployment model with a single network per cluster, but would eventually want to support other cluster and network models.
+
+If you would like to try out ambient multicluster, please follow [this guide](TODO).
+Since many details are in discussion, we would love to hear any of your thoughts, comments, and use cases.
+You can find ways to reach us on the [Istio community page](https://istio.io/latest/about/community/).
```

Comment:
Maybe reduce their clicks by just linking to slack/github? This is also a good spot to stress that this is alpha and NOT ready for production. "If you find a bug, please file it on GitHub" is a good way to end that